index.d.ts

```diff
@@ -241,13 +236,14 @@ export interface PaymentCardTextFieldCommonProps {
   cvcPlaceholder?: string;
   disabled?: boolean;
 
-  onChange?: (params: PaymentCardTextFieldNativeEvent) => void;
+  onChange?(
+    event: ReactNative.NativeSyntheticEvent<PaymentCardTextFieldNativeEvent>
+  ): void;
 
-  /** @deprecated */
-  onParamsChange?: (
+  onParamsChange?(
```

Comment:
changes this to not use the `=>` syntax. It doesn't really allow module augmentation as well as the syntax I changed it to (not sure the name for it)